### ISSUE TITLE
Fragmented mode gives error in piped output

### ISSUE THREAD

**Issue by shubhamchdhary** [gpac/gpac#2492]
Hi @aureliendavid, using https://github.com/gpac/gpac/issues/1625#issuecomment-1581185023, I have compiled GPAC with Kvazaar support. I can now use Kvazaar directly from GPAC (v2.3.0 with the latest FFmpeg). But when trying to use the fragmented mode like this
```
gpac -i video://:fmt=x11grab:framerate=30:video_size=1280x720:dev=: --kvazaar-params=tiles=4x4,slices=tiles,mv-constraint=frametilemargin,input-fps=30,input-res=1280x720 enc:c=libkvazaar tilesplit @ -o named_pipe1:ext=mp4:frag
```

It throws a warning as below
```
[MP4Mux] Cannot add track to already finalized movie in fragmented file, will request a new muxer for that track
No filter chain found for PID video1_tile1 in filter mp4mx to any loaded filters - NOT CONNECTED
```

The generated output file has only one stream, which should be 17 for a 4x4 tiled video, right?
I'm not able to use fragmented mode properly.

**Comment by jeanlf** [comment]
Now fixed on master (b4bb8671)

**Comment by shubhamchdhary** [comment]
Thanks :)
It is working fine now.
